apps/Gruntfile.js

```diff
@@ -438,6 +439,14 @@ module.exports = function (grunt) {
       entries: {
         pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx'
       }
+    },
+
+    {
+      uniqueName: 'publicKeyCryptography',
+      entries: {
+        publicKeyCryptography: './src/publicKeyCryptography/publicKeyCryptography.js'
+      },
+      provides: ['react', 'react-dom', 'radium']
```

Comment:
Oh, it doesn't yet (whoops) but I do expect it to soon - I want to use React to write the widget, and the `publicKeyCryptography` bundle will stand on its own, not requiring the apps common file to work.  Modeled on [the embedBlocks bundle, here](https://github.com/code-dot-org/code-dot-org/pull/9964/files#diff-26cdc4306e5d33ea4df1299321d8852aR422).

Having this here for now doesn't seem to cause any problems - I believe it just prevents aliasing those requires if I were to use them.